libraries/lib-utility/TypeListVisitor.h

```diff
@@ -0,0 +1,41 @@
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file TypeListVisitor.h
+  @brief work in progress
+
+  Paul Licameli
+
+**********************************************************************/
+#ifndef __AUDACITY_TYPELIST_VISITOR__
+#define __AUDACITY_TYPELIST_VISITOR__
+
+#include "TypeList.h"
+#include <type_traits>
+#include <variant>
+
+namespace TypeListVisitor {
+using namespace TypeList; using namespace std;
```

Comment:
I'd love to see `using namespace std` removed here, but no string feelings about it